### ISSUE TITLE
Caravan art

### ISSUE THREAD

**Issue by tukkek** [tukkek/javelin#277]
Surprisingly there hasn't been easy to find an "arabic camel" image for Caravans. Here are the images that have been used so far:

![caravan](https://user-images.githubusercontent.com/3819826/98677753-376f8f00-233c-11eb-9b53-8fc8a1ba0d39.png)  ![caravan](https://user-images.githubusercontent.com/3819826/98677769-42c2ba80-233c-11eb-9f21-81185c8fd134.png)

And a real-life image of what I have in mind:

![caravan](http://img.timeinc.net/time/photoessays/2010/top10_alt_travel/camel_train.jpg)

**Comment by Zireael07** [comment]
https://www.google.com/search?q=pixel+art+camel and click on Images to see some neat references. Nothing usable as-is, unfortunately...

**Comment by tukkek** [comment]
Yeah, I use the Creative Commons function on Google image search a lot and https://opengameart.org but no luck so far! Maybe one of the contributing artists will step up to the task eventually :)

I've been remembering a chat on Discord we had a while ago about equipment in Javelin, which I've now added to the game in an initial version - it's simple but I think it's an interesting take, with a crafting-lite aspect to it. The number is a bit misleading since the system is procedural but earlier I stumbled upon one of the logs and saw the game has 500 different pieces of equipment now!

I've also played around with World enemy spawn rates a lot and made a few of the initial buildings non-capturable which I seem to remember being one of your concerns. Anyway, I hope I'm not misremembering conversations with someone else on Discord since it's been a while but I think it was you lol hopefully once I finally manage to release 1.7  you will give the new features a try!
